apps/src/templates/instructions/ThreeColumns.jsx

```diff
@@ -41,10 +41,10 @@ const ThreeColumns = (props) => {
   };
 
   return (
-    <div style={[styles.container].concat(style)}>
-      <div style={styles.middle}>{children[1]}</div>
-      <div style={styles.left}>{children[0]}</div>
-      <div style={styles.right}>{children[2]}</div>
+    <div style={[styles.container, style.container]}>
+      <div style={[styles.middle, style.middle]}>{children[1]}</div>
+      <div style={[styles.left, style.left]}>{children[0]}</div>
+      <div style={[styles.right, style.right]}>{children[2]}</div>
```

Comment:
It's really hard to visualize distinguish `style` and `styles` here. I wonder if we should actually call the prop something like overrideStyle